src/main/java/org/saltyrtc/client/SaltyRTCServerInfo.java

```diff
@@ -12,4 +13,12 @@
     String getHost(String initiatorPublicKey);
     int getPort(String initiatorPublicKey);
     SSLContext getSSLContext(String initiatorPublicKey);
```

Comment:
We could also add a default method for `getSSLContext` for consistency. (When we started with this lib we were still on Java 7 which doesn't yet support default methods...)